Source/Core/InputCommon/CMakeLists.txt

```diff
@@ -2,7 +2,8 @@ set(SRCS	ControllerEmu.cpp
 			InputConfig.cpp
 			ControllerInterface/ControllerInterface.cpp
 			ControllerInterface/Device.cpp
-			ControllerInterface/ExpressionParser.cpp)
+			ControlReference/ControlReference.cpp
+			ControlReference/ExpressionParser.cpp)
```

Comment:
Okay, I fixed the reference to ExpressionParser and added ControlReference to the same top-level `ControllerInterface` filter. Is that the appropriate thing to do here? I'm not well-versed with Visual Studio.